projects/layout/package.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "@taiga-ui/layout",
+    "version": "3.23.1",
+    "description": "A package with Taiga UI layout components",
+    "keywords": [
+        "angular",
+        "component",
+        "layout"
+    ],
+    "homepage": "https://github.com/tinkoff/taiga-ui",
+    "repository": "https://github.com/tinkoff/taiga-ui",
+    "license": "Apache-2.0",
+    "peerDependencies": {
+        "@angular/common": ">=12.0.0",
+        "@angular/core": ">=12.0.0",
+        "@taiga-ui/cdk": ">=3.23.1",
+        "@taiga-ui/core": ">=3.23.1",
```

```suggestion
        "@taiga-ui/core": ">=3.25.0",
```

(I don't know should we sync it or it will be done automatically)